### PR TITLE
Work on issues

### PR DESCRIPTION
**Pull request by lucasmichot**
Work on milestones

### CODE REVIEW

File: lib/Github/Api/Issue/Comments.php [hunk 1]
```diff
@@ -11,28 +11,42 @@
  */
 class Comments extends AbstractApi
 {
+    /**
+     * Configure the body type.
+     *
+     * @param string $bodyType
```
Comment (GrahamCampbell) [inline lib/Github/Api/Issue/Comments.php]: string|null

File: lib/Github/Api/Issue/Comments.php [hunk 2]
```diff
@@ -11,28 +11,42 @@
...
         }
 
-        $this->client->setHeaders(array($header));
+        $this->client->setHeaders(array(sprintf($header, $this->client->getOption('api_version'))));
     }
 
+    /**
+     * Get all comments for an issue.
+     *
+     * @param        string g$username
```
Comment (GrahamCampbell) [inline lib/Github/Api/Issue/Comments.php]: oops

File: lib/Github/Api/Issue/Comments.php [hunk 3]
```diff
@@ -40,11 +54,29 @@ public function all($username, $repository, $issue, $page = 1)
...
     }
 
+    /**
+     * Create a comment for an issue.
+     *
+     * @param string $username
+     * @param string $repository
+     * @param int    $issue
+     * @param array  $params
+     * @return array
```
Comment (GrahamCampbell) [inline lib/Github/Api/Issue/Comments.php]: spacing wrong in a few places

Comment (GrahamCampbell) [inline lib/Github/Api/Issue/Comments.php]: Should be param, throws, return, each separated by a blank line of phpdoc.

File: lib/Github/Api/Issue/Comments.php [hunk 4]
```diff
@@ -11,28 +11,42 @@
...
+     * Configure the body type.
+     *
+     * @param string $bodyType
+     */
     public function configure($bodyType = null)
     {
         switch ($bodyType) {
             case 'raw':
-                $header = sprintf('Accept: application/vnd.github.%s.raw+json', $this->client->getOption('api_version'));
+                $header = 'Accept: application/vnd.github.%s.raw+json';;
```
Comment (stof) [inline lib/Github/Api/Issue/Comments.php]: you have a duplicated semi-colon here

File: lib/Github/Api/Issue/Comments.php [hunk 5]
```diff
@@ -11,28 +11,42 @@
...
                 break;
 
             case 'text':
-                $header = sprintf('Accept: application/vnd.github.%s.text+json', $this->client->getOption('api_version'));
+                $header = 'Accept: application/vnd.github.%s.text+json';
```
Comment (stof) [inline lib/Github/Api/Issue/Comments.php]: the variable does not store a header anymore. It will be a proper header only after the sprintf call.

**Pull request merged**